src/credentials/sso/ssoAccessTokenProvider.ts

```diff
@@ -77,26 +86,29 @@ export class SsoAccessTokenProvider {
                     startUrl: this.ssoUrl,
                     region: this.ssoRegion,
                     accessToken: tokenResponse.accessToken!,
-                    expiresAt: new Date(Date.now() + tokenResponse.expiresIn! * MILLISECONDS_PER_SECOND).toISOString(),
+                    expiresAt: new Date(this.currentTimePlusSecondsInMs(tokenResponse.expiresIn!)).toISOString(),
                 }
                 return accessToken
             } catch (err) {
                 if (err.code === 'SlowDownException') {
-                    retryInterval += BACKOFF_TIME
+                    retryInterval += BACKOFF_DELAY_MINUTES
                 } else if (err.code === 'AuthorizationPendingException') {
                     // do nothing, wait the interval and try again
                 } else if (err.code === 'ExpiredTokenException') {
+                    getLogger().error(err)
```

Comment:
Nit this (and the `getLogger` statement below) will log the error three times (possibly more?): once here, once in `accessToken()`and once in `refreshCredentials()` (since you're rethrowing the same error each time).

Reply:
These all will now bubble up to refreshCredentials